### PR TITLE
xautoclaim

### PR DESCRIPTION
**Pull request by AvitalFineRedis**
### Pull Request check-list

_Please make sure to review and check all of these items:_

- [x] Does `$ tox` pass with this change (including linting)?
- [x] Do the CI tests pass with this change (enable it first in your forked repo and wait for the github action build to finish)?
- [x] Is the new or changed code fully tested?
- [x] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?

_NOTE: these things are not required to open a PR and can be done
afterwards / while the PR is open._

### Description of change

support xautoclaim command

### CODE REVIEW

**Comment by AvitalFineRedis** [comment]
@chayim

File: redis/client.py [hunk 1]
```diff
@@ -2578,6 +2585,42 @@ def xadd(self, name, fields, id='*', maxlen=None, approximate=True,
...
+        justid: optional boolean, false by default. Return just an array of IDs
+        of messages successfully claimed, without returning the actual message
+        """
+        if not isinstance(min_idle_time, int) or min_idle_time < 0:
+            raise DataError("XAUTOCLAIM min_idle_time must be a non negative"
+                            " integer")
+        kwargs = {}
+        pieces = [name, groupname, consumername, str(min_idle_time), start_id]
+
+        if count is not None:
```
Comment (chayim) [inline redis/client.py:2612]:
```suggestion
        if count is not None and not isinstance(count, int)
```

**Comment by chayim** [comment]
@AvitalFineRedis Instead of force passing ints around - I'd rely on the untyped nature of python. Cast the value of the string to an int and see what happens. Double casting an int in python is effectively free.

**Review by chayim — CHANGES_REQUESTED** [review]
@AvitalFineRedis Instead of force passing ints around - I'd rely on the untyped nature of python. Cast the value of the string to an int and see what happens. Double casting an int in python is effectively free.

**Pull request merged**